pathfinding_service.rst

```diff
@@ -95,7 +95,10 @@ The arguments are POSTed as a JSON object.
 
 Returns
 """""""
-A list of path objects. A path object consists of the following information:
+
+A list of path objects and a feedback token when successful.
+
+Each path object consists of the following information:
 
```

Comment:
is the content of the table what the path object consists of? So each path object is a list of tuples of a path and the corresponding fee. This is what we call path object, right? And for every path object we now have a feedback token?

Reply:
No, one feedback token is valid for all paths from a certain request.